udata_front/theme/gouvfr/assets/js/components/MarkdownEditor/Milkdown/TableTooltip/TableTooltip.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div class="fr-hidden">
+    <div class="fr-grid-row" ref="toolTipRef">
+      <TooltipButton
+        v-if="!isWholeTable && !isHeading && isRow"
+        icon="fr-icon-arrow-up-line"
+        class="fr-mr-1v"
+        @click="() => {
+          if (loading) return;
+
+          getEditor()?.action((ctx) => {
+            ctx.get(commandsCtx).call(addRowBeforeCommand.key);
+          });
+          tooltipProvider?.hide();
+        }"
+      />
+      <TooltipButton
+        v-if="!isWholeTable && isCol"
+        icon="fr-icon-arrow-left-line"
+        class="fr-mr-1v"
+        @click="() => {
+          if (loading) return;
+
+          getEditor()?.action((ctx) => {
+            ctx.get(commandsCtx).call(addColBeforeCommand.key);
+          });
+          tooltipProvider?.hide();
+        }"
+      />
+      <TooltipButton
+        v-if="isWholeTable || (!isHeading && isAny)"
+        icon="fr-icon-delete-line"
+        class="fr-mr-1v"
+        @click="() => {
+          if (loading) return;
+
+          getEditor()?.action((ctx) => {
+            ctx.get(commandsCtx).call(deleteSelectedCellsCommand.key);
+          });
+          tooltipProvider?.hide();
+        }"
+      />
+
+      <TooltipButton
+        v-if="!isWholeTable && isRow"
+        icon="fr-icon-arrow-down-line"
+        class="fr-mr-1v"
+        @click="() => {
+          if (loading) return;
+
+          getEditor()?.action((ctx) => {
+            ctx.get(commandsCtx).call(addRowAfterCommand.key);
+          });
+          tooltipProvider?.hide();
+        }"
+      />
+      <template v-if="!isWholeTable && isCol">
+        <TooltipButton
+          icon="fr-icon-arrow-right-line"
+          class="fr-mr-1v"
+          @click="() => {
+            if (loading) return;
+            getEditor()?.action((ctx) => {
+              ctx.get(commandsCtx).call(addColAfterCommand.key);
+            });
+
+            tooltipProvider?.hide();
+          }"
+        />
+        <TooltipButton
+          :svg="AlignLeftIcon"
+          class="fr-mr-1v"
+          @click="() => {
+            if (loading) return;
+            getEditor()?.action((ctx) => {
+              ctx.get(commandsCtx).call(setAlignCommand.key, 'left');
+            });
+          }"
+        />
+        <TooltipButton
+          class="fr-mr-1v"
+          :svg="AlignCenterIcon"
+          @click="() => {
+            if (loading) return;
+            getEditor()?.action((ctx) => {
+              ctx.get(commandsCtx).call(setAlignCommand.key, 'center');
+            });
+          }"
+        />
+        <TooltipButton
+          class="fr-mr-1v"
+          :svg="AlignRightIcon"
+          @click="() => {
+            if (loading) return;
+            getEditor()?.action((ctx) => {
+              ctx.get(commandsCtx).call(setAlignCommand.key, 'right');
+            });
+          }"
+        />
```

Comment:
I don't think I saw labels for these icons. Their roles are not obvious at first glance, maybe add some title or label on hover?

Reply:
Adding `title` could be great for consistency too